journalbeat/README.md

```diff
@@ -0,0 +1,27 @@
+# journalbeat
+
+Journalbeat is a lightweight shipper for journald logs with Elasticsearch.
+
+## Maintainers
+
+The Habitat Maintainers (humans@habitat.sh).
```

Comment:
Super ultra minor thing. Can the email here go in `< >` brackets? 🙂

Reply:
Found a rogue space at the end of line 7, removed it and reformatted the email address.